.travis.yml

```diff
@@ -16,5 +15,5 @@ services:
 install:
  - npm install
 
-before_install:
- - npm install -g grunt-cli
+before_script:
+ - npm run lint
```

Comment:
Great to add lint to the .travis.yml checks.

However, it seems no specific fix in this PR has been done regarding linting. Maybe I have miss it, maybe the lint script is not working properly or maybe the code was already perfect from the point of linting :) What do you think?

Just to be sure...

Reply:
Great. NTC